src/register/RegistrationPage.test.jsx

```diff
@@ -299,6 +306,31 @@ describe('RegistrationPage', () => {
       });
     });
 
+    it('should submit form without UsernameField when autoGeneratedUsernameEnabled is true', () => {
```

Comment:
can we add one more test that username field is not displayed when `ENABLE_AUTO_GENERATED_USERNAME` is true?

Reply:
Sure Added.